mxnet/inference/docker/1.9/py3/cu112/Dockerfile.gpu

```diff
@@ -31,7 +31,7 @@ LABEL dlc_major_version="1"
 
 ARG PYTHON_VERSION=3.8.10
 ARG MMS_VERSION=1.1.8
-ARG MX_URL=https://aws-mx-pypi.s3.us-west-2.amazonaws.com/1.9.0/aws_mx_cu112-1.9.0b20211227.163-py3-none-manylinux2014_x86_64.whl
+ARG MX_URL=https://framework-binaries.s3.us-west-2.amazonaws.com/mxnet/v1.9.0_telemetry/aws_mx_cu112-1.9.0b20221123.1-py3-none-manylinux2014_x86_64.whl
```

Comment:
I see that the new binaries have been built in a different s3 bucket. Is there a reason for the same?

Reply:
We've been using this new bucket for awhile now. 
https://github.com/aws/deep-learning-containers/commit/db6977c607cc85f0af65ecd78a537a41e513dc34#diff-fe0edbe322492286efa96fdf55c6133c0c9cfd103f0b7406a775945cf5448bdf

It's also used for TF. https://github.com/aws/deep-learning-containers/blob/master/tensorflow/training/docker/2.11/py3/Dockerfile.cpu#L192